### ISSUE TITLE
yolo v2 loss function bug?

### ISSUE THREAD

**Issue by bceulema** [thtrieu/darkflow#855]
On [line 71 of yolov2/train.py](https://github.com/thtrieu/darkflow/blob/b2aee0000cd2a956b9f1de6dbfef94d53158b7d8/darkflow/net/yolov2/train.py#L71)

This seems to be following the loss function for yolo v1 instead of yolo v2.
Shouldn't you use the [adjusted_coords_wh](https://github.com/thtrieu/darkflow/blob/b2aee0000cd2a956b9f1de6dbfef94d53158b7d8/darkflow/net/yolov2/train.py#L61) instead?

**Comment by rlustig-gpfw** [comment]
@bceulema [This](https://github.com/thtrieu/darkflow/blob/b2aee0000cd2a956b9f1de6dbfef94d53158b7d8/darkflow/net/yolov2/train.py#L61) line shows that `adjusted_coords_wh` is being used as expected.
